platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -130,6 +130,8 @@ public interface CoreI18n extends EmfI18n {
 		.de("Klicken, um die Programm-Logs anzuzeigen.");
 	I18n0 CLIENT_IP_ADDRESS = new I18n0("Client IP Address")//
 		.de("Client IP Adresse");
+	I18n0 COLLAPSE_NAVIGATION_FOLDERS = new I18n0("Collapse Navigation Folders")//
```

Comment:
```suggestion
	I18n0 AUTOMATICALLY_COLLAPSE_FOLDERS = new I18n0("Automatically Collapse Folders")//
```
Since this is displayed in a "Navigation"-titled fieldset, we could omit "Navigation" from these identifiers... kind of redundant. Also, I think that adding "Automatically" adds some clarity here.
(DB field name could remain as-is, btw)

Reply:
Db name needs to change aswell since the auto generation forces this